packages/shared/src/components/modals/LegacyOnboardingModal.tsx

```diff
@@ -58,7 +58,7 @@ const getFooterButton = ({
 };
 
 interface LegacyOnboardingModalProps extends ModalProps {
```

Comment:
Perhaps also typecast the target_type here, just so we 100% it aligns

Reply:
Oh, sorry. I think I did some cleanups and the `target_type` was not needed so I removed it already.